nebula/ui/components/inAppAuth/VPNInAppAuthenticationInputs.qml

```diff
@@ -46,24 +46,42 @@ ColumnLayout {
 
         spacing: VPNTheme.theme.listSpacing
 
-        VPNTextField {
-            id: textInput
-            objectName: base.objectName + "-textInput"
+        RowLayout {
             Layout.fillWidth: true
-            _placeholderText: _inputPlaceholderText
-            Keys.onReturnPressed: col.submitInfo(textInput)
-            onDisplayTextChanged: if (hasError) hasError = false
-        }
+            spacing: VPNTheme.theme.windowMargin / 2
 
-        VPNPasswordInput {
-            id: passwordInput
-            objectName: base.objectName + "-passwordInput"
-            _placeholderText: _inputPlaceholderText
-            Layout.fillWidth: true
-            Keys.onReturnPressed: col.submitInfo(passwordInput)
-            onTextChanged: if (hasError) hasError = false
+            VPNTextField {
+                id: textInput
+                objectName: base.objectName + "-textInput"
+                Layout.fillWidth: true
+                _placeholderText: _inputPlaceholderText
+                Keys.onReturnPressed: col.submitInfo(textInput)
+                onDisplayTextChanged: if (hasError) hasError = false
+            }
+
+            VPNPasswordInput {
+                id: passwordInput
+                objectName: base.objectName + "-passwordInput"
+                _placeholderText: _inputPlaceholderText
+                Keys.onReturnPressed: col.submitInfo(passwordInput)
+                Layout.fillWidth: true
+                onTextChanged: if (hasError) hasError = false
+            }
+
+            VPNPasteButton {
+                id: inputPasteButton
+                objectName: base.objectName + "-inputPasteButton"
+                Layout.preferredWidth: VPNTheme.theme.rowHeight
+                Layout.preferredHeight: VPNTheme.theme.rowHeight
+                height: undefined
+                width: undefined
```

Comment:
I don't think you have to specifically undefine these (even though they are defined in VPNIconButton) because Layout.prefferedWidth/Height should completely overwrite them